pkg/features/kube_features.go

```diff
@@ -212,6 +212,15 @@ const (
 	// Enables support for time zones in CronJobs.
 	CronJobTimeZone featuregate.Feature = "CronJobTimeZone"
 
+	// owner: @thockin
+	// deprecated: v1.28
```

Comment:
oh, this is much better than current method we were doing

Reply:
I see https://github.com/kubernetes/kubernetes/pull/117696#discussion_r1189210781

Reply:
Yeah, I didn't realize this was an option either :)